jenkins/test/testsuite/Jenkinsfile

```diff
@@ -14,6 +16,11 @@ pipeline {
                 script {
                     properties([
                             parameters([
```

Comment:
Let's take the parameters block out of a stage just like in https://github.com/opensearch-project/opensearch-build/pull/1099